### PR TITLE
Plugins documentation

### PR DESCRIPTION
**Pull request by sdirosa**
This PR introduces new categories in each SDK to list which Plugins are available, enabled by default, optional, etc. 
- [x] Angular SDK
- [x] Browser SDK
- [x] React SDK
- [x] React Native SDK

Extras:
- Some fixes to React Native docs examples
- Some renaming in Angular docs, it was still referring to Browser Tracker instead of Browser SDK

### CODE REVIEW

**Review by ivarpruijn — APPROVED** [review]
Two mini spelling suggestions, but lgtm. Let's merge and deploy today.

File: docs/docs/tracking/angular/introduction.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Introduction
 ---
 # Angular SDK
 
-The Angular SDK is a small Angular Module, that comes with an Angular Directive, built on top of Browser Tracker.   
+The Angular SDK is a small Angular Module, that comes with an Angular Directive, built on top of the Browser SDK.
```
[the remark this answers is not in the document]
Comment (sdirosa) [reply docs/docs/tracking/angular/introduction.md:8]: Let's NEVER use relative links please. They are impossible to grep and make for a really annoying maintenance experience.

File: docs/docs/tracking/react-native/plugins/react-navigation.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
+# ReactNavigation
+
+Optional plugin, for anyone using ReactNavigation, providing automatic tracking of RootLocationContext, PathContext, Screen listeners for Tabs and Drawers and an automatically tracked Link components.
```
[the remark this answers is not in the document]
Comment (sdirosa) [reply docs/docs/tracking/react-native/plugins/react-navigation.md]: Thanks, committed!

File: docs/docs/tracking/angular/introduction.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ivarpruijn) [reply docs/docs/tracking/angular/introduction.md:8]: Didn't know that, okay! I'll update the suggestion.

[the remark this answers is not in the document]
Comment (sdirosa) [reply docs/docs/tracking/angular/introduction.md:8]: If you want to find easily where a certain link is used globally in the docs, good luck doing that with relative links ;)

With absolute ones:
1. Docusaurus link checker works 
2. Links are absolutely unique across the whole documentation so you can easily search for them
3. IDEs can follow and validate links for you and also "refactor" them for you, at least PyCharm does

[the remark this answers is not in the document]
Comment (ivarpruijn) [reply docs/docs/tracking/angular/introduction.md]: Makes sense, thanks, will use absolute links from now on

**Comment by ivarpruijn** [comment]
I'll merge this now @sdirosa to prepare an update on staging

File: docs/docs/tracking/angular/introduction.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sdirosa) [reply docs/docs/tracking/angular/introduction.md]: At least for the Tracker section it would be much appreciated, thanks! 🙏🏼

**Pull request merged**